### PR TITLE
Agribalyse proxies

### PR DESCRIPTION
**Pull request by teolemon**
Agribalyse proxies

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -60941,6 +60949,9 @@ fi:sardiineja oliiviöljyssä
 fr:Sardines à l'huile d'olive, sardines à l'huile d'olive vierge, sardines à l'huile d'olive vierge extra, sardines à l'huile d'olive égouttée
 nl:Sardientjes op olijfolie
 ro:Sardine în ulei de măsline
+agribalyse_proxy_food_code:en:26231
+agribalyse_proxy_food_name:en:26231
```
Comment (teolemon) [inline taxonomies/categories.txt:60953]:
```suggestion
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -60941,6 +60949,9 @@ fi:sardiineja oliiviöljyssä
 fr:Sardines à l'huile d'olive, sardines à l'huile d'olive vierge, sardines à l'huile d'olive vierge extra, sardines à l'huile d'olive égouttée
 nl:Sardientjes op olijfolie
 ro:Sardine în ulei de măsline
+agribalyse_proxy_food_code:en:26231
```
Comment (teolemon) [inline taxonomies/categories.txt:60952]:
```suggestion
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -60941,6 +60949,9 @@ fi:sardiineja oliiviöljyssä
...
+agribalyse_proxy_food_name:en:26231
+agribalyse_proxy_food_name:fr:European pilchard or sardine, fillets without fishbone, in olive oil, canned, drained
```
Comment (teolemon) [inline taxonomies/categories.txt:60955]:
```suggestion
ro:Sardine în ulei de măsline
agribalyse_proxy_food_code:en:26231
agribalyse_proxy_food_name:fr:European pilchard or sardine, fillets without fishbone, in olive oil, canned, drained

```

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -2666,6 +2666,14 @@ nl:Roséciders
...
 <en:Ciders
 en:Flavoured ciders
+fr:Cidres aromatisés
+
+<en:Flavoured ciders
+en:Cherry flavoured ciders
+fr:Cidres aromatisés cerise
+
+<en:Flavoured ciders
+en:Strawberry flavoured ciders
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:2676]:
```suggestion
en:Raspberry flavoured ciders
```

Comment (teolemon) [reply taxonomies/categories.txt:2676]: fixed

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -60941,6 +60949,8 @@ fi:sardiineja oliiviöljyssä
 fr:Sardines à l'huile d'olive, sardines à l'huile d'olive vierge, sardines à l'huile d'olive vierge extra, sardines à l'huile d'olive égouttée
 nl:Sardientjes op olijfolie
 ro:Sardine în ulei de măsline
+agribalyse_proxy_food_code:en:26231
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:60952]: There are exact matches for this category, it should be merged with the ciqual category that was added:

<en:Sardines in olive oil
<en:Sardine fillets
en:European pilchard or sardine in olive oil canned drained
fr:Sardine à l'huile d'olive appertisée égouttée
agribalyse_food_code:en:26040
ciqual_food_code:en:26040
ciqual_food_name:en:European pilchard or sardine, in olive oil, canned, drained

Comment (stephanegigandet) [reply taxonomies/categories.txt:60952]: (same for all the ciqual categories that just add "canned, drained")

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**